src/exodus/walletmodels.h

```diff
@@ -83,31 +77,42 @@ class SigmaMintId
 class SigmaMint
 {
 public:
-    PropertyId property;
-    DenominationId denomination;
+    SigmaMintId id;
```

Comment:
This will increase a chance to have error for key mismatch due to `SigmaMintId` contains public key. We should generate it only when we really need it.